src/loader.ts

```diff
@@ -73,8 +94,9 @@ export default class {
      * Checks if the source file was modified, if it is, then `build()` is executed to rebuild the changed source file.
      */
     async ifSourceModify() {
-        const lm = lastModified(this.config.importPath);
-        const lmfile = `${this.cwd}/@types/${basename(this.config.importPath)}/lastModified`;
+        const lm = lastModified(this.config.importPath),
+            lmfile = `${this.cwd}/@types/${basename(this.config.importPath)}/lastModified`;
```

Comment:
This is a chaotic change and takes away readability. I would suggest reverting them.